src/panels/lovelace/editor/config-elements/hui-generic-editor.ts

```diff
@@ -0,0 +1,82 @@
+import { CSSResultGroup, html, LitElement, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { fireEvent } from "../../../../common/dom/fire_event";
+import { capitalizeFirstLetter } from "../../../../common/string/capitalize-first-letter";
+import { LocalizeFunc } from "../../../../common/translations/localize";
+import "../../../../components/ha-form/ha-form";
+import type { HaFormSchema } from "../../../../components/ha-form/types";
+import { LovelaceCardConfig } from "../../../../data/lovelace";
+import type { HomeAssistant } from "../../../../types";
+import type { LovelaceGenericElementEditor } from "../../types";
+import { configElementStyle } from "./config-elements-style";
+
+@customElement("hui-generic-editor")
+export class HuiGenericEditor
```

Comment:
Should we name this `hui-form-editor` to make it more descriptive? And match `getConfigForm` better?